.github/workflows/go.yml

```
@@ -7,7 +7,7 @@ on:
     branches: main
 
 env:
-  GOLANG: 1.18
+  GOLANG: 1.20
```

Took me a while to figure out and seems quite silly, but the errors we were getting was because the release is 1.20.3 and not 1.2
- https://go.dev/dl/
- https://github.com/actions/go-versions/releases
 
```suggestion
  GOLANG: 1.20.3
```